src/Semigroupoids/Do.hs

```diff
@@ -0,0 +1,91 @@
+{-# LANGUAGE CPP #-}
+
+#if __GLASGOW_HASKELL__ >= 702
+{-# LANGUAGE Safe #-}
+#endif
+
+#if __GLASGOW_HASKELL__ == 708
+{-# OPTIONS_GHC -fno-warn-amp #-}
+#endif
+
+{-|
+
+This module re-exports operators from 'Data.Functor.Apply' and 'Data.Functor.Bind', but under the same
```

Comment:
Haddock uses double quotes to render module names, whereas single quotes are used for specific identifiers:

```suggestion
This module re-exports operators from "Data.Functor.Apply" and "Data.Functor.Bind", but under the same
```